py_clob_client/clob_types.py

```diff
@@ -78,7 +78,13 @@ class MarketOrderArgs:
 
     amount: float
```

Comment:
we should specify what units (collateral vs condition token) the amount are in for market args in a doc string here imo.

Reply:
This field has some docs annotations below
<img width="431" alt="image" src="https://github.com/user-attachments/assets/93072719-451e-4a55-89ad-1e6bb1e7f4d6" />

do you think it is enough?

Reply:
ohy eah that's fine - kind of missed that - whoops

Reply:
tbh: this was causing me some headaches - and i am not sure if this should be expected behavior!!!